metadeploy/labels_en.json

```diff
@@ -9,16 +9,12 @@
             "description": "tagline of product"
         },
         "description": {
-            "message": "Welcome to the Outbound Funds Module installer!\r\n\r\nThe Outbound Funds Module gives you the tools to easily manage and track funding requests and outgoing funds.  \r\n\r\n*IMPORTANT*: Salesforce does not recommend installing upgrades directly in production organizations. Instead, install Outbound Funds Module in one of your free sandbox environments or a developer organization first.\r\n\r\nOnce you've installed Outbound Funds Module, check out resources on getting started in our documentation:\r\n\r\n* [Outbound Funds Module Documentation](https://powerofus.force.com/s/article/OFM-Documentation)\r\n\r\nQuestions? Become a member of the [Outbound Funds Module](https://powerofus.force.com/s/group/0F980000000CvlMCAS) community group in the Power of Us Hub.",
+            "message": "Welcome to the Outbound Funds Module installer!\r\n\r\nThe Outbound Funds Module gives you the tools to easily manage and track funding requests and outgoing funds.  \r\n\r\n*IMPORTANT*: Salesforce does not recommend installing upgrades directly in production organizations. Instead, install Outbound Funds Module in one of your free sandbox environments or a developer organization first.\r\n\r\nOnce you've installed Outbound Funds Module, check out resources on getting started in our documentation:\r\n\r\n* [Outbound Funds Module Documentation](https://powerofus.force.com/s/article/OBF-Documentation)\r\n\r\nQuestions? Become a member of the [Outbound Funds Module](https://powerofus.force.com/s/group/0F980000000CvlMCAS) community group in the Power of Us Hub.",
             "description": "shown on product detail page (markdown)"
         },
         "error_message": {
             "message": "To get help with this error, go to [help.salesforce.com](https://help.salesforce.com/), find the Support & Services area, and log a technical support ticket with Salesforce. Include \u201cOutbound Funds Module\u201d in the subject line and include in your comment the installation link shown here.",
             "description": "shown after failed installation (markdown)"
-        },
-        "click_through_agreement": {
```

Comment:
@SalesforceFoundation/release-engineering-reviewers :

The translation team generated these translation files located in the `translation-integration/2020-06` branch. 
[translation-integration/2020-06/metadeploy/labels_en.json](https://github.com/SalesforceFoundation/OutboundFunds/blob/translation-integration/2020-06/metadeploy/labels_en.json) does NOT include a `click_through_agreement` for any `metadata` translations.  Is this correct?

Reply:
We don't want to keep either of the diff hunks in `labels_en.json`. The version in `master` is correct, and was unfortunately modified after the initial translation drop.

Reply:
This is definitely a little confusion but as I understand it: 
1) I believe we are safe to merge this en_US `click_through_agreement` from master into the feature/translations branch. It was not included in the `labels_en.json` file because we didn't want it to go to the translators. Legal will be providing their own translations.
2) Once legal does provide their translations, we'll need to add them to the other labels_xx files.
@davidmreed, @dmkud  Did I get this correct?

Reply:
Exactly, @dvenor.